psamm/datasource/sbml.py

```diff
@@ -872,12 +894,20 @@ def write_model(self, file, model):
             species_tag.set(
                 self._sbml_tag('compartment'),
                 model_compartments[species.compartment])
+            species_tag.set(self._sbml_tag('constant'), 'true')
+            species_tag.set(self._sbml_tag('boundaryCondition'), 'true')
+            species_tag.set(self._sbml_tag('hasOnlySubstanceUnits'), 'true')
+
+        if self._cobra_flux_bounds is False:
```

Comment:
Actually, maybe we should always generate `listOfParameters` since this can now coexist with the COBRA-style local parameters (right?).